server/db/model/Application.js

```diff
@@ -3,24 +3,32 @@ var mongoose = require('../index.js'), User = require('./User.js');
 // Define the document Schema
 var schema = new mongoose.Schema({
     user: String,
+    birthday: Date,
+    university: String,
+    major: String,
+    tshirt_size: {
+        type: String,
+        enum: ['xs', 's', 'm', 'l', 'xl', '2xl', '3xl']
+    },
     experience: {
         type: String,
-        enum: ['veteran', 'novice']
-    }
+        enum: ['novice', 'experienced', 'veteran']
+    },
+    resume: String
 });
 
 // Allow us to query by token
 schema.query.byToken = function(findToken) {
-    return Promise(function(resolve, reject) {
+    const Application = this;
+
+    return new Promise(function(resolve, reject) {
         User.find()
             .byToken(findToken)
             .exec()
             .then(user => {
-                resolve(
-                    this.findOne({
-                        user: user.email
-                    })
-                );
+                const app = Application.findOne({ user: user.email });
+                console.log('app', app);
```

Comment:
Don't necessarily think we need to define the constant before return the query function